translator/trace/oc_const.go

```diff
@@ -1,3 +1,17 @@
+// Copyright 2019 OpenTelemetry Authors
```

Comment:
nit: 2020 but I guess we need to update the tool

Reply:
Done manually, please update the tool if you know where it is.

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/604